draft-ietf-rats-corim.md

```diff
@@ -1120,22 +1115,44 @@ the object relates to the subject.
 
 #### Device Identity Triple {#sec-comid-triple-identity}
 
-A Device Identity triple relates one or more cryptographic keys to a device.
-The subject of an Identity triple uses an instance or class identifier to refer
-to a device, and a cryptographic key is the object. The predicate asserts that
-the identity is authenticated by the key. A common application for this triple
-is device identity.
+A Device Identity triple record relates one or more cryptographic keys to a device identity.
+The cryptographic keys are bound to or associated with a Target Environment that is within the device.
+The device identifier may be part of the Target Environment's `environment-map` or may be part of some other device identity credential, such as a certificate.
+The cryptographic keys are expected to be used to authenticate the device.
+
+Device Identity triples instruct a Verifier to perform key validation checks, such as revocation, certificate path construction & verification, or proof of possession.
+The Verifier SHOULD perform offline verification of keys contained in Device Identity triples.
+
+A Device Identity triple endorses that the keys were securely provisioned to the named Target Environment.
+Additional details about how a key was provisioned or is protected may be asserted using Endorsements such as  `endorsed-triples`.
+
+Depending on key formatting, as defined by `$crypto-key-type-choice`, the Verifier may take different steps to locate and verify the key.
+
+If a key has usage restrictions that limit its use to device identity challenges, Verifiers SHOULD check for key use that violates usage restrictions.
+
+Offline verification of keys or verification of key use restrictions MAY produce Claims that are added to the ACS.
```

Comment:
We already know that a Verifier can add claims of whatever sort. What measurement-value-map code point are you talking about for these? If there is nothing specific, I think this is too vague to be a useful addition.